### ISSUE TITLE
[Infrastructure] Research & determine whether Helm charts should be an interim solution

### ISSUE THREAD

**Issue by jessicadaugherty** [pokt-network/pocket#298]
## Objective

Considering we have an ongoing k8s operator project for v1, determine if we should use Helm chart to power v1 deployments as an interim solution or migrate to Kubernetes Operators. Ideally this ticket should be done in parallel with https://github.com/pokt-network/pocket/issues/299

## Origin Document

@olshansky

## Goals

- [ ] Determine if we should use Helm charts now and Kubernetes Operators in the future, or if KO should be utilized in lieu of Helm charts

## Deliverables

- [ ] Evaluate Helm charts as an interim solution
- [ ] Evaluate the ease of implementation of Kubernetes Operators
- [ ] Define a path forward 

## Non-goals / Non-deliverables

- TBD

## General issue checklist
- [ ] Update the appropriate CHANGELOG
- [ ] Update the README
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)
- [ ] Update any relevant global documentation & references
- [ ] Document small issues / TODOs along the way
---

**Creator**: @jessicadaugherty

**Comment by Olshansk** [comment]
Discussed this in the team meeting. I believe we aligned that the goal is to replace LocalNet deployment with k8s operators in the 2-week iteration starting this Wednesday.

Does that sound reasonable @okdas?

**Comment by jessicadaugherty** [comment]
Can this be closed @Olshansk?

**Comment by jessicadaugherty** [comment]
@okdas did a lot of research and determined not to use Helm charts
